test/unit/test_materials.py

```diff
@@ -297,3 +297,14 @@ def test_error_wrong_solubility_law_string():
         match="Acceptable values for solubility_law are 'henry' and 'sievert'",
     ):
         F.Material(1, 1, 1, solubility_law="foo")
+
+
+def test_materials_setter():
+    my_model = F.Simulation()
+    test_materials = F.Materials([])
+    my_model.materials = test_materials
+    assert my_model.materials == test_materials
```

Comment:
This shouldn't really be in test_materials since it's not testing functionality in the Materials class but rather the simulation class

Reply:
Moved this